#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE_NAME = "squirrel_pointcloud_filter"

gen = ParameterGenerator()
gen.add("object_frame", str_t, 0, "", "object_link")
gen.add("enabled", bool_t, 0, "", True)
gen.add("use_const_object_dim", bool_t, 0, "", True)
gen.add("object_topic", str_t, 0, "", "object_size")
gen.add("size_x", double_t, 0, "", 0.25, 0.0, 10.0)
gen.add("size_y", double_t, 0, "", 0.25, 0.0, 10.0)
gen.add("size_z", double_t, 0, "", 0.25, 0.0, 10.0)

exit(gen.generate(PACKAGE_NAME, "squirrel_pointcloud_filter", "GraspFilter"))